packages/api/tests/lib/test-granule-delete.js

```diff
@@ -467,40 +407,18 @@ test.serial(
         }))
     );
 
-    // Add granule to elasticsearch
-    const esGranulesClient = new Search(
-      {},
-      'granule',
-      process.env.ES_INDEX
-    );
-
-    await t.context.esClient.client.index({
-      index: t.context.esIndex,
-      type: 'granule',
-      id: newGranule.granuleId,
-      parent: 'fakeCollectionId',
-      body: newGranule,
-    });
-
-    await deleteGranuleAndFiles({
+    await t.throwsAsync(deleteGranuleAndFiles({
       knex: t.context.knex,
-      apiGranule: newGranule,
       pgGranule: undefined,
-      esClient: t.context.esClient,
-    });
+    }), { message: 'pgGranule undefined, is required' });
 
-    // verify the files are deleted from S3.
+    // verify the files are not deleted from S3, since deleteGranule errored
     await Promise.all(
       files.map(async (file) => {
-        t.false(await s3ObjectExists({ Bucket: file.bucket, Key: file.key }));
+        t.true(await s3ObjectExists({ Bucket: file.bucket, Key: file.key }));
       })
     );
 
-    // Verify record is removed from elasticsearch
-
-    const esResult = await esGranulesClient.get(newGranule.granuleId);
-    t.is(esResult.detail, recordNotFoundString);
-
     t.teardown(() =>
       deleteS3Buckets([s3Buckets.protected.name, s3Buckets.public.name]));
   }
```

Comment:
The test title:
test.serial(
  'deleteGranuleAndFiles() does not require a PostgreSQL granule if an elasticsearch granule is present'
I don't think we have this test case any more.